src/kit/Section.tsx

```diff
@@ -27,7 +34,9 @@ const Section: React.FC<Props> = ({ children, titleDivider = false, title }) =>
           <h5 className={titleClasses.join(' ')}>{title}</h5>
         </div>
       )}
-      <div className={css.body}>{children}</div>
+      <div className={css.body} id="section-body">
```

Comment:
do we need this `id` attribute?

Reply:
Good catch, no. That got left over from some testing.